tests/scripts/all.sh

```diff
@@ -390,6 +393,7 @@ pre_parse_command_line () {
             --armcc) no_armcc=;;
             --armc5-bin-dir) shift; ARMC5_BIN_DIR="$1";;
             --armc6-bin-dir) shift; ARMC6_BIN_DIR="$1";;
+            --error-test) error_test=$((error_test + 1));;
```

Comment:
Could you clarify why do you add +1, instead of assigning error_test = 1?

Reply:
This way you can pass it multiple times, if you want to simulate having multiple failed components.

Reply:
I don't get what would be the reason to call `pseudo_component_error_test` more than once.

Reply:
I used that to check the reporting when multiple components fail.

It can also detect if the component is not being run in a subshell: the `error_test` variable would be kept from one invocation to the next.